Tasks/app-store-promote/task.json

```diff
@@ -65,6 +65,34 @@
             "helpMarkDown": "The password of the account used to publish to the Apple App Store.",
             "visibleRule": "authType = UserAndPass"
         },
+                {
+            "name": "isTwoFactorAuth",
+            "type": "boolean",
+            "label": "Use Two-Factor Authentication",
```

Comment:
At https://appleid.apple.com/account/manage, Apple refers to "Two-step Verification."  As a compromise, should we be calling this "two-step authentication" instead of "two-factor authentication," to align more closely with their terminology?